curations/maven/mavencentral/com.sun.mail/javax.mail.yaml

```diff
@@ -4,6 +4,9 @@ coordinates:
   provider: mavencentral
   type: maven
 revisions:
+  1.5.0:
```

Comment:
The license information is incorrect, which was provided by mistake, pls reject the PR, thanks.